backend/cn/lib/source_injection.ml

```diff
@@ -422,37 +441,39 @@ let output_injections oc cn_inj =
       Ok ())
     ()
 
-
-open Cerb_frontend
-
-let get_magics_of_statement stmt =
-  let open AilSyntax in
-  let rec aux acc (AnnotatedStatement (_loc, Annot.Attrs xs, stmt_)) =
-    let acc =
-      List.fold_left
-        (fun acc attr ->
-          let open Annot in
-          match (attr.attr_ns, attr.attr_id, attr.attr_args) with
-          | Some (Symbol.Identifier (_, "cerb")), Symbol.Identifier (_, "magic"), xs ->
-            List.map (fun (loc, str, _) -> (loc, str)) xs :: acc
-          | _ -> acc)
-        acc
-        xs
-    in
-    match stmt_ with
-    | AilSskip | AilSexpr _ | AilSbreak | AilScontinue | AilSreturnVoid | AilSreturn _
-    | AilSgoto _ | AilSdeclaration _ | AilSreg_store _ ->
-      acc
-    | AilSblock (_, ss) | AilSpar ss -> List.fold_left aux acc ss
-    | AilSif (_, s1, s2) -> aux (aux acc s1) s2
-    | AilSwhile (_, s, _)
-    | AilSdo (s, _, _)
-    | AilSswitch (_, s)
-    | AilScase (_, s)
-    | AilScase_rangeGNU (_, _, s)
-    | AilSdefault s
-    | AilSlabel (_, s, _)
-    | AilSmarker (_, s) ->
-      aux acc s
-  in
-  aux [] stmt
+(* This appears to be unused:
```

Comment:
Delete if so (in a separate PR).

Reply:
In principle I also like deleting unused stuff which is unused, but since this code is in development, I wasn't sure if this is perhaps something that we intended to use, but is not yet plugged in.